server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -194,6 +194,8 @@ static TransportVersion def(int id) {
     public static final TransportVersion NODE_STATS_REQUEST_SIMPLIFIED = def(8_561_00_0);
     public static final TransportVersion TEXT_EXPANSION_TOKEN_PRUNING_CONFIG_ADDED = def(8_562_00_0);
 
+    public static final TransportVersion ESQL_ASYNC_QUERY = def(8_999_00_0);
```

Comment:
yeah. I just added this as a placeholder so as to avoid some conflicts - should have added a comment saying such. Let's fix it to a real version now, and update as necessary (as the PR progresses)